frontend/app/src/people/utils/AssignedUnassignedBounties.tsx

```diff
@@ -129,6 +156,13 @@ const Bounties = (props: BountiesProps) => {
   const { ui } = useStores();
   return (
     <>
+      {props.uuid && props.name && (
```

Comment:
instead of doing props.uuid and props.name above we restructure props, if you can add uuid and name to that it would be cleaner code